csv-data-management/info.json

```diff
@@ -0,0 +1,68 @@
+{
+    "name": "csv-data-management",
+    "version": "1.0.0",
+    "label": "CSV Data Management",
+    "description": "",
+    "publisher": "",
+    "icon_small_name": "fortisoar_logo.png",
+    "icon_large_name": "fortisoar_logo.png",
+    "category": "",
+    "configuration": {
+        "fields": []
```

Comment:
remove this completely. it will, not otherwise show the message that the connector doesn't require a configuration

Reply:
Done